pkg/access/commands/use.go

```diff
@@ -0,0 +1,61 @@
+package commands
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/apono-io/apono-cli/pkg/aponoapi"
+)
+
+const outputFlagName = "output"
+
+func AccessDetails() *cobra.Command {
+	var outputFormat string
+
+	cmd := &cobra.Command{
+		Use:     "use [id]",
+		GroupID: Group.ID,
+		Short:   "List all access sessions",
+		Args:    cobra.ExactArgs(1), // This will enforce that exactly 1 argument must be provided
+		RunE: func(cmd *cobra.Command, args []string) error {
+			client, err := aponoapi.GetClient(cmd.Context())
+			if err != nil {
+				return err
+			}
+
+			id := args[0]
+			accessDetails, _, err := client.ClientAPI.AccessSessionsAPI.GetAccessSessionAccessDetails(cmd.Context(), id).Execute()
+			if err != nil {
+				return err
+			}
+
+			var output string
+			switch outputFormat {
+			case "cli":
+				output = accessDetails.GetCli()
+			case "link":
+				link := accessDetails.GetLink()
+				output = link.GetUrl()
+			case "instructions":
+				output = accessDetails.Instructions
+			case "json":
+				outputBytes, _ := json.Marshal(accessDetails)
+				output = string(outputBytes)
+			}
```

Comment:
ofir raised some interesting point for the cli
if the default output is cli, and it's a command,
maybe by default we want to run it, and have a flag for "--no-auto-run"
or maybe have default to show but a flag for "--auto-run"

Reply:
not sure i understand